de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/account/XMPPAccount.java

```diff
@@ -76,6 +77,11 @@
         this.useTLS = useTLS;
     }
 
+    public void sonarViolation() throws Exception {
+      // Should cause a github comment from sonarqube
+      throw new Exception("FAIL");
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Define and throw a dedicated exception instead of using a generic one. ([squid:S00112](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS00112))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=102&issues=AWP9mQNa3e-qcckj_oH8&open=AWP9mQNa3e-qcckj_oH8)

[ISSUE_KEY]: AWP9mQNa3e-qcckj_oH8